### ISSUE TITLE
HDR constrains FPS to 60

### ISSUE THREAD

**Issue by RecklessBoon** [TheElixZammuto/moonlight-xbox#109]
**Describe the bug**
When HDR 10 is enabled, the fps is constrained to 60 fps when 120 fps is set in the configuration.

**To Reproduce**
1. Go to Device settings
2. Click on 120 fps value and HDR enabled and save
3. Connect to device
4. Check TV reported FPS

**Expected behavior**
Once connected, the TV/monitor should report 120 fps with HDR content being used instead of 60 fps with HDR content

** Logs **
While pressing Menu + View and then "Show Logs", are there any relevant log lines?
TBD

**Guest Information:**
 - Xbox Model: [Xbox Series X]
 - TV Resolution: [4K HDR @120FPS]

**Host Information**
- Operating System: [Windows]
- Host Software: [Sunshine]
- Monitor Resolutions: [4K HDR @120fps]

**Additional context**
This was found while testing v1.15.6 in #107

**Comment by RecklessBoon** [comment]
After the recent update to 1.16, I can't get HDR to work at all now.

**Comment by RecklessBoon** [comment]
With 1.16.2, HDR works again. Seems to be stuck at 60 fps again though.

**Comment by TheElixZammuto** [comment]
Yup I unfortunately had to revert that specific change. I'll test again soon with the correct display and deploy 1.16.3 which should work as expected at least at 60fps and hopefully at 120fps too

**Comment by TheElixZammuto** [comment]
Has this been fixed with version 1.17.0?
